frontend/src/libs/components/modal/styles.module.scss

```diff
@@ -0,0 +1,56 @@
+.modal {
```

Comment:
Note for wide screens: I believe we should make modal width same as component inside it. Otherwise there will be a lot of empty space.